tornado-runtime/src/main/java/uk/ac/manchester/tornado/runtime/tasks/PrebuiltTask.java

```diff
@@ -246,4 +247,11 @@ public int[] getAtomics() {
         return atomics;
     }
 
+    public Class<?> getClassJAR() {
+        return klass;
+    }
+
+    public void withKlassJAR(Class<?> classJar) {
+        this.klass = classJar;
+    }
```

Comment:
I am not sure about the `classJar` and `getClassJAR`, `withKlassJAR` names. To my understanding you pass the name of the class file (e.g. `MyKlass.class`) and not the jar, right? it is assumed that the jar must be in the class file.

Is this correct?

Reply:
We pass any class that appears in the JAR file that we want to load